features/support/paths.rb

```diff
@@ -467,6 +467,18 @@ def path_to(page_name, *args)
     when 'the dns settings page'
       admin_site_dns_path
 
+    when 'the spam protection page'
+      edit_admin_site_spam_protection_path
+
+    when 'the xss protection page'
+      edit_admin_site_developer_portal_path
+
+    when 'the sso integrations page'
+      provider_admin_authentication_providers_path
```

Comment:
Kinda the same as https://github.com/3scale/porta/pull/175/files#r226311032
This already exists in:
https://github.com/3scale/porta/blob/23b4398ad7a97fb4b0a66f112deca50c7280203b/features/support/paths.rb#L25-L26